packages/cli/tests/build.test.js

```diff
@@ -1,7 +1,8 @@
 const { join } = require('path');
```

Comment:
This test suite was not instilling much confidence in me, hence all these changes.

I will be extracting this all out into a separate test suite PR, to ensure this one doesn't get out of hand / bogged down in irrelevant work, but I wanted to stage it here so I get a better idea of where we're at while I build it out.

...and I want to delay rebasing this monster for as along as I can.